src/consensus/params.h

```diff
@@ -34,6 +34,7 @@ enum DeploymentPos : uint16_t {
     DEPLOYMENT_TESTDUMMY,
     DEPLOYMENT_CHECKTEMPLATEVERIFY, // Deployment of CTV (BIP 119)
     DEPLOYMENT_ANYPREVOUT,
+    DEPLOYMENT_OPCAT,
```

Comment:
Should be `OP_CAT` as pointed out here: https://github.com/bitcoin-inquisition/bitcoin/pull/39/files#r1491766392